scripts/adminPage.js

```diff
@@ -1,9 +1,10 @@
+//TODO: DONE! Go through all js files and comment above each code block what the code is doing to help ensure both comprehension and readability for later examination.
 'use strict';
 Testimonial.renderAdminPage = function() {
-  var source = $('#testimonial-template').html();
+  var source = $('#admin-template').html();
   var awesomeRender = Handlebars.compile(source);
   Testimonial.awesomePointsByAuthor().forEach(function(awesomePointsObj) {
-    $('.awesome-points ul').append(awesomeRender(awesomePointsObj));
+    $('#admin ul').append(awesomeRender(awesomePointsObj));
   });
   $('.testimonial-total span').text(Testimonial.allTestimonials.length);
 };
```

Comment:
This is a really clever way to tackle the .reduce() problem we were faced with before. If you're looking for another challenge, I bet you could add drop down menus that would let the user sort testimonials by author...